consul/hooks/run

```diff
@@ -3,6 +3,7 @@
 exec 2>&1
 
 SERVERMODE={{cfg.server.mode}}
+export CONSUL_UI_BETA={{cfg.server.beta_ui}}
```

Comment:
So for this to work, we actually don't need to specify the "export". Just remove that and it will work great.

Reply:
When I have `CONSUL_UI_BETA=true` in my plan I still get the old UI when I start Consul.

I have to add `export` to be able to view the new UI.

Reply:
That's because your repo is using the old version. Can you please rebase with the latest Consul repo to get the 1.1.0 version? Try that and you should be good :)